src/datasets/load.py

```diff
@@ -1004,7 +1005,15 @@ def __init__(
 
     def get_module(self) -> DatasetModule:
         readme_path = os.path.join(self.path, "README.md")
+        standalone_yaml_path = os.path.join(self.path, ".huggingface.yml")
```

Comment:
i think it's .yaml on the hub...

Reply:
Good catch, I got confused by an internal PR name

Reply:
never trust an internal PR 🤣